script.js

```diff
@@ -0,0 +1,527 @@
+const load_button = document.getElementById('load-button');
+const back_button = document.getElementById('back-button');
+const search_button_name = document.getElementById('search-button-name');
```

Comment:
```suggestion
const searchButtonName = document.getElementById('search-button-name');
```

El estándar para JavaScript es camel case, snake case no es recomendado (sobretodo por manejo de archivos json)